spec/models/contact_form_spec.rb

```diff
@@ -0,0 +1,95 @@
+
+
+#  `rails generate curation_concerns:work GenericWork`
+
+require 'rails_helper'
+
+describe ContactForm do
+
+  describe "it requires input" do
+
+    before do
+
+      subject.contact_method = "Contact Menthod"
+      subject.category = "Category"
+      subject.name = "John Smith"
+      subject.email = "abc@123.com"
+      subject.subject = "This is the subject"
+      subject.message = "This is a message."
+
+    end
+
+
+    it "validates contact_method" do
+
+      subject.contact_method = nil
+      expect(subject).to be_valid
```

Comment:
I don't think contact method is presently used by the form, so it can be
nil.  Make sense?

On Thu, Mar 24, 2016 at 8:47 PM, Colin <notifications@github.com> wrote:

> In spec/models/contact_form_spec.rb
> <https://github.com/mlibrary/umrdr/pull/278#discussion_r57408907>:
>
> > +    before do
> > +
> > +      subject.contact_method = "Contact Menthod"
> > +      subject.category = "Category"
> > +      subject.name = "John Smith"
> > +      subject.email = "abc@123.com"
> > +      subject.subject = "This is the subject"
> > +      subject.message = "This is a message."
> > +
> > +    end
> > +
> > +
> > +    it "validates contact_method" do
> > +
> > +      subject.contact_method = nil
> > +      expect(subject).to be_valid
>
> Is there a validation on .contact_method? Is it supposed to let it benil`?
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mlibrary/umrdr/pull/278/files/7bc84d3906b4051719f26c397d1696dc0351b527#r57408907>
>